### PR TITLE
sms audio

### PR DESCRIPTION
**Pull request by kdsolo**
incoming/outgoing audio SMS added

### CODE REVIEW

File: apps/system/js/haiku_emoji_panel.js [hunk 1]
```diff
@@ -61,6 +61,10 @@
       var target = e.target;
       var receiver = window.app.pairNumber;
       var messageBody = target && target.dataset.icon;
+      var smsTone = document.querySelector('#sms_tone');
+      smsTone.play();
```
Comment (punamdahiya) [inline apps/system/js/haiku_emoji_panel.js:65]: Do we need to play smsTone here?

Comment (sfoster) [reply apps/system/js/haiku_emoji_panel.js:65]: I wondered that too...

**Comment by sfoster** [comment]
Merged and landed as-is in: e28f1f03c4548dc77a67fa130883264ba10423d3

File: apps/system/js/haiku_emoji_panel.js (hunk 1 above)
Comment (kdsolo) [reply apps/system/js/haiku_emoji_panel.js:65]: Good catch...I believe this should be moved to "flashEmojis"

**Pull request closed without merging**